packages/augur-core/source/libraries/NetworkConfiguration.ts

```diff
@@ -2,187 +2,192 @@ import { ethers } from 'ethers';
+                : new ethers.utils.BigNumber(process.env.ETHEREUM_GAS_LIMIT)
+    },
+    kovan: {
+        isProduction: false,
+        http:
+            'https://eth-kovan.alchemyapi.io/jsonrpc/1FomA6seLdWDvpIRvL9J5NhwPHLIGbWA',
+        privateKey: process.env.ETHEREUM_PRIVATE_KEY,
+        gasPrice: (typeof process.env.ETHEREUM_GAS_PRICE_IN_NANOETH === 'undefined'
+                ? new ethers.utils.BigNumber(20)
+                : new ethers.utils.BigNumber(process.env.ETHEREUM_GAS_PRICE_IN_NANOETH!)
+        ).mul(new ethers.utils.BigNumber(1e9)),
+        gasLimit:
+            typeof process.env.ETHEREUM_GAS_LIMIT === 'undefined'
+                ? new ethers.utils.BigNumber(7500000)
+                : new ethers.utils.BigNumber(process.env.ETHEREUM_GAS_LIMIT)
+    },
+    rinkeby: {
+        isProduction: false,
+        http:
+            'https://eth-rinkeby.alchemyapi.io/jsonrpc/Kd37_uEmJGwU6pYq6jrXaJXXi8u9IoOM',
+        ws: 'wss://rinkeby.augur.net/ethereum-ws',
+        privateKey: process.env.ETHEREUM_PRIVATE_KEY,
+        gasPrice: (typeof process.env.ETHEREUM_GAS_PRICE_IN_NANOETH === 'undefined'
+                ? new ethers.utils.BigNumber(20)
+                : new ethers.utils.BigNumber(process.env.ETHEREUM_GAS_PRICE_IN_NANOETH!)
+        ).mul(new ethers.utils.BigNumber(1e9)),
+        gasLimit:
+            typeof process.env.ETHEREUM_GAS_LIMIT === 'undefined'
+                ? new ethers.utils.BigNumber(6900000)
+                : new ethers.utils.BigNumber(process.env.ETHEREUM_GAS_LIMIT)
+    },
+    clique: {
+        isProduction: false,
+        http: 'http://clique.ethereum.nodes.augur.net',
+        privateKey:
+            process.env.ETHEREUM_PRIVATE_KEY ||
+            'fae42052f82bed612a724fec3632f325f377120592c75bb78adfcceae6470c5a',
+        gasPrice: new ethers.utils.BigNumber(1),
+        gasLimit:
+            typeof process.env.ETHEREUM_GAS_LIMIT === 'undefined'
+                ? new ethers.utils.BigNumber(7500000)
+                : new ethers.utils.BigNumber(process.env.ETHEREUM_GAS_LIMIT)
+    },
+    environment: {
+        isProduction: process.env.PRODUCTION === 'true' || false,
+        http: 'http://localhost:8545',
+        ws: 'ws://localhost:8546',
+        privateKey:
+            process.env.ETHEREUM_PRIVATE_KEY ||
+            'fae42052f82bed612a724fec3632f325f377120592c75bb78adfcceae6470c5a',
+        gasPrice: (typeof process.env.ETHEREUM_GAS_PRICE_IN_NANOETH === 'undefined'
+                ? new ethers.utils.BigNumber(20)
+                : new ethers.utils.BigNumber(process.env.ETHEREUM_GAS_PRICE_IN_NANOETH!)
+        ).mul(new ethers.utils.BigNumber(1000000000)),
+        gasLimit:
+            typeof process.env.ETHEREUM_GAS_LIMIT === 'undefined'
+                ? new ethers.utils.BigNumber(7500000)
+                : new ethers.utils.BigNumber(process.env.ETHEREUM_GAS_LIMIT),
+        gnosisRelayerAddress: '0x9d4c6d4b84cd046381923c9bc136d6ff1fe292d9',
```

Comment:
I also formatted this file so here I'll point out this line is the most relevant.